Desenvolver Protótipo de Baixa Fidelidade

Rocsantos opened the pull request:
Utilizando o programa Balsamiq Wireframe (https://balsamiq.com/), Pedro Haick e eu, Rodrigo Santos, desenvolvemos o Protótipo de Baixa Qualidade.

Co-Authored-By: peHaick <pedrohaickzz@gmail.com>

Rocsantos committed:
Desenvolver Protótipo de Baixa Fidelidade

Utilizando o programa Balsamiq Wireframe (https://balsamiq.com/), Pedro Haick e eu, Rodrigo Santos, desenvolvemos o Protótipo de Baixa Qualidade.

Co-Authored-By: peHaick <57498006+peHaick@users.noreply.github.com>

GeovanaRamos reviewed (changes requested):
Pessoal, é preciso que vcs vinculem as imagens na nossa documentação para que fique visível em https://fga-eps-mds.github.io/2020.1-Grupo4. Para isso façam:
- Coloquem a pasta do prototipo dentro da pasta **produto**, e nao documentos. Use o nome prototipo_baixa para a pasta, pra facilitar futuras referências.
- Adicionem um arquivo prototipo_baixa.md na pasta produto, escrevam um pouco sobre o que foi feito, adicionem versionamento e então as imagens com a opção de zoom do md.
- Adicionem em mkdocs.yml uma referencia na nav bar com o projeto de vcs `- Protótipo de Baixa Fidelidade: produto/prototipo_baixa.md`.
- Testem as alterações como já instruímos vocês e está detalhado no Readme.


Vou deixar os comentários específicos de conteúdo das imagens com a @gabiMSilva. Só avisando que tem uma foto que não está em PNG. Já façam a troca.

Rocsantos committed:
Mover a pasta do protótipo de documentos, para produto.
Renomear as imagens para facilitar coloca-las no arquivo prototipo_baixa.md
Excluir o arquivo .bpmr, que era o arquivo que o Balsamiq gera no projeto do protótipo
Criar o arquivo prototipo_baixa.md, onde será explicado cada imagem do Protótipo de Baixa Fidelidade
Alterar o arquivo mkdocs.yml para inserir o prototipo_baixa.md

Co-authored-by: peHaick <pedrohaickzz@gmail.com>
